model-optimizer/mo/front/mxnet/extractor.py

```diff
@@ -50,20 +50,17 @@ def common_mxnet_fields(node: Node):
     }
 
 
-def mxnet_op_extractor(node: Node):
+def mxnet_op_extractor(node: Node, lowered_keys_map: dict):
```

Comment:
Before the change the MxNet op extractor was case sensitive. Now it is case insensitive. Did we check that this doesn't break anything? @iimironov, could you comment on that? Do we have MxNet ops which types are distinct with case only?

Reply:
Yes we have. For simple example you can look at SwapAxis extractor.